src/api_wrappers/generatePythonAPIWrappers.py

```diff
@@ -109,8 +109,8 @@ def camel_case_to_underscore(name):
 for method in json.loads(sys.stdin.read()):
     route, signature, opts = method
     wrapper_method_name = camel_case_to_underscore(signature.split("(")[0])
-    retry = "True" if (opts['retryable']) else "False"
-    accept_nonce = True if 'acceptNonce' in opts else False
+    accept_nonce = opts.get('acceptNonce', False)
+    retry = opts['retryable'] or accept_nonce
```

Comment:
The way it's done for now is, all nonce-taking routes are marked as **not** retryable in wrapper_table.json, and then this line makes it so that they are actually eligible for retry, in these bindings (so, exactly the situation you are asking about, I think). That means there are sort of three classes of routes: not retryable (`retryable=false`), retryable with a nonce (`retryable=false` in the wrapper table but it's marked as retryable in the bindings), and retryable (`retryable=true`).

If at some point we decide that all language bindings are required to implement the nonce support correctly, then we could remove this special logic here and just mark all nonce-taking routes as actually retryable.